index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css">
+  <title>Project 4: Pizzabot</title>
+</head>
+
+<body>
+  <section class="box">
+    <h1>This is your Pizza Bot</h1>
+    <img src="media/pizza.png" class="picture">
+    <p>You can choose Hawaiian Pizza, Pepperoni Pizza or Vegetarian Pizza and you can have as many as you want</p>
+    <form action="" method="get" class="form">
+      <label> Write the Pizza you want
+        <input type="text" id="pType" name="pizzaKind" class="input" placeholder="Pepperoni Pizza">
+      </label>
+      <label> Write the number of pizzas
+        <input type="text" id="pNum" name="pizzaNum" class="input" placeholder="3">
+      </label>
+    </form>
+    <p class="word" id=word1>You have yet order are you new?</p>
+    <button class="button" id="button1">Order here</button>
```

Comment:
Maybe a more describing naming convention?
```suggestion
    <button class="button" id="order-button">Order here</button>
```